app/views/menus/show.html.haml

```diff
@@ -0,0 +1,14 @@
+%h1= "Menu for #{@facade.menu.day_stamp}"
+%h2 Orders:
+%ul
+  = render @facade.orders
+
+= render 'orders/form'
```

Comment:
Don't use instance variables inside of partials, pass them as locals, e.g
`render 'orders/form', menu: @facade.menu, order: @facade.new_order`
This will allow you to reuse them in different places